letsencrypt/DOCS.md

```diff
@@ -373,6 +373,55 @@ transip_api_key: ''
 
 </details>
 
+<details>
+  <summary>RFC2136</summary>
+
+  You will need to set up a server with RFC2136 (Dynamic Update) support with a TKEY (to authenticate the updates).  How to do this will vary depending on the DNS server software in use.  For Bind9, you first need to first generate an authentication key by running
+  ```
+  # dnssec-keygen -a HMAC-SHA512 -b 512 -n HOST letsencrypt
+  Kletsencrypt.+165+20675
+  ```
+  The key file (Kletsencrypt.+165+20675.key in this example) looks like the following:
```

Comment:
There should be a blank line around code blocks
https://github.com/markdownlint/markdownlint/blob/master/docs/RULES.md#md031---fenced-code-blocks-should-be-surrounded-by-blank-lines

Reply:
Code that are expected to be executed should generally not have a prefix <https://developers.home-assistant.io/docs/documenting/create-page#embedding-code>
Pasting it with `#` would not do anything, if separation is needed for command/output, use `$`

Reply:
Okay, pushed a change that fixes both of these issues.
Anything else you see?
Thanks!